Admin/CategoryAdmin.php

```diff
@@ -60,15 +64,15 @@ protected function configureFormFields(FormMapper $formMapper)
         $formMapper
             ->with('General', array('class' => 'col-md-6'))
                 ->add('name')
-                ->add('description', 'textarea', array(
+                ->add('description', TextareaType::class, array(
```

Comment:
It's not that simple, this has to stay 2.3-compatible. See other bundles for making this conditional.